SConstruct

```diff
@@ -6,6 +6,7 @@ EnsureSConsVersion(0, 98, 1)
 import glob
 import os
 import sys
+import pickle
```

Comment:
Please keep alphabetical order for the imports.

Reply:
Now you have pickle twice :)

Reply:
what a pickle! should be fixed